src/navigationDrawer/navigationDrawer.component.tsx

```diff
@@ -158,7 +159,32 @@ class NavigationDrawer extends Component<CombinedNavigationProps> {
   }
 
   public render(): React.ReactElement {
-    const imgSrc = this.props.darkMode ? STFCLogoWhiteText : STFCLogoBlueText;
+    let navDrawerLogo = this.props.darkMode
+      ? STFCLogoWhiteText
+      : STFCLogoBlueText;
+
+    let altTxt = getString(this.props.res, 'alternative-text');
+
+    if (this.props.navigationDrawerLogo) {
+      if (this.props.darkMode) {
+        if (this.props.navigationDrawerLogo.dark) {
```

Comment:
If you use my suggestion of error checking the settings file to ensure both logos are defined if the overall logo config is defined, you can remove these if statements